.github/workflows/pull-request.yml

```diff
@@ -176,6 +180,40 @@ jobs:
       - run: pnpm test:silent
         working-directory: ${{ env.EDITOR_DIRECTORY }}
 
+  build_localplanning_services:
```

Comment:
Pretty much copy/pasted from how the React Editor app is built and handled, probably some room here to simplify. Worth revisiting after more progress is made on the monorepo work.